.github/workflows/publish_navigation_component_ado.yml

```diff
@@ -0,0 +1,83 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: publish-navigation-component-ado
+# This workflow builds and publishes in ADO the Foldable Navigation Component artifacts (all of them).
+
+on:
+  push:
+    tags:
+      - 'dualscreen*'
+
+  workflow_dispatch:
+    inputs:
+      name:
+        description: 'Triggers publication to ADO - Foldable Navigation Component'
+      home:
+        description: 'location'
+        required: false
+
+jobs:
+  publish-dualscreenSDK:
```

Comment:
Please, change this name, e.g. `publish-NavigationComponent`

Reply:
Done